pkg/identity/identity.go

```diff
@@ -30,6 +32,17 @@ type Identity struct {
 	LabelsSHA256 string `json:"labelsSHA256"`
 }
 
+// IPIdentityPair is a pairing of an IP and the security identity to which that
+// IP corresponds.
+//
+// WARNING - STABLE API
+// This structure is written as JSON to the key-value store. Do NOT modify this
+// structure in ways which are not JSON forward compatible.
```

Comment:
I don't see JSON formatting expressions after the field declarations below.. should there be?